data-vis/testing_wisconsin-covid.Rmd

```diff
@@ -99,6 +98,21 @@ p <-
 
 ggplotly(p, tooltip = c("text"))
 
+
+plot_data %>% 
+  mutate(cases = ifelse(cases==0, NA, cases)) %>% 
```

Comment:
Interesting... it works locally. Apparently, st_coordinates.sfc does not know what to do with "sfc_GEOMETRY". 
Try to add 
    st_cast("MULTIPOLYGON") %>% 
after the mutate statement.
This also works locally.

Reply:
Got it to work by updating `sf` and `plotly` packages to the most recent version.  Thanks!